package.json

```
@@ -22,6 +22,7 @@
     "prepare": "husky install",
     "test": "npm run test:unit && npm run test:e2e",
     "test:unit": "jest --config ./jest.unit.config.js ./src",
+    "test:unit:watch": "jest --config ./jest.unit.config.js ./src --coverage --watch",
```

How about referring to existing script assuming we want to add run `test:unit` with additional options:
```suggestion
    "test:unit:watch": "npm run test:unit -- --coverage --watch",
```

---

Great suggestion, will modify.